File: VB/WindowsApplication3/ArrowClickEventArgs.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars
Imports DevExpress.XtraBars.Styles
Imports DevExpress.XtraBars.ViewInfo
Imports DevExpress.XtraBars.Painters

Namespace WindowsApplication3
	Public Class ArrowClickEventArgs
		Inherits EventArgs

		Public Sub New()
		End Sub


		'// Fields...

		Private _Link As BarItemLink
		Private _Handled As Boolean

		Public Property Handled() As Boolean
			Get
				Return _Handled
			End Get
			Set(ByVal value As Boolean)
				_Handled = value
			End Set
		End Property


		Public Property Link() As BarItemLink
			Get
				Return _Link
			End Get
			Set(ByVal value As BarItemLink)
				_Link = value
			End Set
		End Property

	End Class
End Namespace

File: VB/WindowsApplication3/ArrowClickEventArgs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;
using DevExpress.XtraBars.Styles;
using DevExpress.XtraBars.ViewInfo;
using DevExpress.XtraBars.Painters;

namespace WindowsApplication3
{
    public class ArrowClickEventArgs : EventArgs
    {

        public ArrowClickEventArgs() { }


        //// Fields...

        private BarItemLink _Link;
        private bool _Handled;

        public bool Handled
        {
            get { return _Handled; }
            set
            {
                _Handled = value;
            }
        }


        public BarItemLink Link
        {
            get { return _Link; }
            set { _Link = value; }
        }

    }
}
